File: The simple interactive story creator/theme/js/data-classes/story.js
import { textToId } from "../text-utility.js";

export const constructGeneral = (title, preview, font, author, authorId, keepPrivate)=>{
    return {
        id: textToId(title),
        title: title,
        preview: preview,
        font: font,
        author: author,
        authorId: authorId,
        keepPrivate: keepPrivate
    };
};

File: The simple interactive story/Assets/Scripts/SimpleStory/Data/Story.cs
namespace SimpleStory.Data
{
    public class Story
    {
        public string id;
        public string title;
        public string description;
        public string authorId;
        public string author;
        public string banner;
        public string publish;
        public string creatingDate;
        public string font;
        
        public override string ToString()
        {
            return "Story:\n" +
                   $"id: {id}\n" +
                   $"title: {title}\n" +
                   $"description: {description}\n" +
                   $"authorId: {authorId}\n" +
                   $"author: {author}\n" +
                   $"banner: {banner}\n" +
                   $"publish: {publish}\n" +
                   $"creatingDate: {creatingDate}\n" +
                   $"font: {font}\n";
        }
    }
}